src/node/rpc/handler_registry.h

```diff
@@ -39,6 +42,17 @@ namespace ccf
       MayWrite
     };
 
+    /** A Handler represents a user-defined endpoint that can be invoked by
+    * authorised users via HTTP requests, over TLS. A Handler is accessible at a
+    * specific verb and URI, e.g. POST /app/accounts or GET /app/records.
+    *
+    * Handlers can read from and mutate the state of the replicated key-value
+    store. The effects of Handler execution are handled by the underlying
```

Comment:
`The effects of Handler execution ....` maybe tmi here?

Reply:
Yeah, I agree 👍